test/second_q/operators/test_fermionic_op.py

```diff
@@ -89,18 +119,41 @@ def test_compose(self):
             )
             self.assertEqual(fer_op, targ)
 
+        with self.subTest("multi compose with parameters"):
+            fer_op = FermionicOp({"+_0 +_1 -_1": self.a, "-_0 +_0 -_1": 1}) @ FermionicOp(
+                {"": 1, "-_0 +_1": self.b}
+            )
+            fer_op = fer_op.simplify()
+            targ = FermionicOp(
+                {
+                    "+_0 +_1 -_1": self.a,
+                    "-_0 +_0 -_1": 1,
+                    "+_0 -_0 +_1": self.a * self.b,
```

Comment:
```suggestion
                    "+_0 +_1 -_0": -self.a * self.b,
```
After #893 the order of terms is preserved. Once #900 is merged, it will also compute the sign correctly.

Reply:
Done.